std/json.d

```diff
@@ -230,6 +230,43 @@ struct JSONValue
         return store.floating;
     }
 
+    /***
+     * Value getter/setter for boolean stored in JSON
```

Comment:
Missing full stop at the end of the sentence.

Reply:
Done